community/kernel/src/main/java/org/neo4j/kernel/impl/api/store/StorePropertyCursor.java

```diff
@@ -78,14 +80,15 @@
     private int offsetAtBeginning;
     private int blocks;
     private int remainingBlocksToRead;
-    private long header;
     private PropertyType type;
     private int keyId;
     private ByteBuffer buffer;
 
     private AbstractDynamicStore.DynamicRecordCursor recordCursor;
     private long currentRecordId;
-    private long[] valuesAfterHeader;
+
+    // in values[0] will be placed the property block header
+    private long[] values = new long[VALUES_SIZE];
```

Comment:
it is easier this way when you need to copy all those data into a byte buffer, to make it nicer and avoid confusion we can have a small method called `header()` which will give you the right data from that array. WDYT?